tests/python/test_blueprint.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# MIT License
+#
+# Copyright (c) 2021 TROUVERIE Joachim <jtrouverie@joakode.fr>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import unittest
+from http import HTTPStatus
+
+from flask import Blueprint, Flask
+
+from flask_inertia import Inertia, render_inertia
+from flask_inertia.unittest import InertiaTestResponse
+
+
+class TestConfig:
+
+    TESTING = True
+    INERTIA_TEMPLATE = "base.html"
+
+
+def index():
+    return render_inertia("Index")
+
+
+class TestBlueprint(unittest.TestCase):
+    """Flask-Inertia blueprint tests."""
```

Comment:
I added a new test class (and test file), but if you'd prefer a different setup let me know. There was some copy/pasting from `test_app.py`.

Reply:
I'm not against adding a new file to tests Blueprint but I would prefer to move the tests setup in the `__init__` file instead of copying it.

Reply:
Do you mean making a base class in `__init__.py` which `TestInertia` and `TestBlueprint` would inherit from?

Reply:
Not necessarily a base class but maybe all the views and  the config class to avoid copy/pasting.